collateral/HiC_QC_report_template.md

```diff
@@ -0,0 +1,48 @@
+# Hi-C library QC report
```

Comment:
Looks nice and will give an easy template to make the HTML version template.  Great work on all of this!

Reply:
it is actually writing an HTML file which renders and has the plots and everything as an intermediate step. there is also a style.css in collateral/. so it could be taken as is.

Reply:
Even better.  Using jinja or something else?

Reply:
just markdown.